src/test/kotlin/JiraPerformanceComparisonIT.kt

```diff
@@ -43,7 +43,7 @@ class JiraPerformanceComparisonIT {
         val results: List<EdibleResult> = AbruptExecutorService(newCachedThreadPool()).use { pool ->
             listOf(
                 benchmark("a.properties", JiraDcScenario::class.java, quality, pool),
-                benchmark("equifax.properties", JiraCloudScenario::class.java, quality, pool)
```

Comment:
The file names were anonymous for a reason. Please, be more careful. FYI @ysemenkov